pontoon/homepage/static/css/homepage.css

```diff
@@ -316,10 +312,12 @@ h2 {
 
 #section-6 .footer .github {
   text-align: center;
+  font-size: 14px;
 }
 
 #section-6 .footer .contact {
   text-align: right;
+  font-size: 14px;
```

Comment:
It would be simpler and easier to maintain if you added this property to `#section-6 .footer`.